components/feature/qr/src/main/java/mozilla/components/feature/qr/QrFragment.kt

```diff
@@ -197,6 +210,13 @@ class QrFragment : Fragment() {
 
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         textureView = view.findViewById<View>(R.id.texture) as AutoFitTextureView
+        customViewFinder = view.findViewById<View>(R.id.view_finder) as CustomViewFinder
+        scanMessage?.let {
+            MainScope().launch {
```

Comment:
Can you add a comment here why we're launching? Presumably because the `customViewFinder` isn't drawn yet?